plugins/modules/cloud/misc/proxmox_interfaces.py

```diff
@@ -0,0 +1,414 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2021, Andreas Botzner (@botzner_andreas) <andreas at botzner dot com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = r'''
+---
+module: proxmox_interfaces
+short_description: Management network interfaces on a Proxmox node.
+version_added: 3.5.0
+description:
+  - Allows you to create/update/delete network interfaces of a Proxmox node.
+author: "Andreas Botzner (@botzner_andreas) <andreas at botzner dot com>"
+options:
+  config:
+    description: Interface configuration
+    required: true
+    type: list
+    elements: dict
+    suboptions:
+      name:
+        description:
+          - Name of the network interface.
+        type: str
+        required: true
+      type:
+        description:
+          - Network interface type
+        type: str
+        default: bridge
+        choices:
+        - bridge
+        - bond
+        - eth
+        - alias
+        - vlan
+        - OVSBridge
+        - OVSBond
+        - OVSPort
+        - OVSIntPort
+        - unknown
+      address:
+        description:
+          - IPv4 address of the interface
```

Comment:
```suggestion
          - IPv4 address of the interface.
```
Descriptions should always have complete sentences (except short_description). Please also fix the other ones.